analyzers/tests/SonarAnalyzer.Test/TestCases/SecureRandomSeedsShouldNotBePredictable.cs

```diff
@@ -267,4 +269,98 @@ void SecureRandom_Arrays(byte b, char[] whoKnows)
         sr.SetSeed(xs.Cast<byte>().ToArray());
         sr.Next(); // Compliant FN, array is predictable
     }
+
+    void FromArguments(SecureRandom sr, VmpcRandomGenerator rng1, DigestRandomGenerator rng2, int seed, byte[] bs)
+    {
+        sr.Next(); // Compliant
+        sr.SetSeed(SEED);
+        sr.Next(); // Compliant
+        sr.SetSeed(seed);
+        sr.Next(); // Compliant
```

Comment:
This is mixing multiple tests. You need to test these cases separately:
- unknown sr
- unknown sr with predictable seed
- predictable sr with unknown seed